README.md

```diff
@@ -101,17 +101,16 @@ console.log(executionResult);
 
 Compiles the `document` AST, using an optional operationName and compiler options.
 
-- `schema` {GraphQLSchema} - `graphql-js` schema object
-- `document` {DocumentNode} - document query AST ,can be obtained by `parse` from `graphql-js`
-- `operationName` {string} - optional operation name in case the document contains multiple queries/operations.
-- `compilerOptions` {Object} - Configurable options on the agent pool
+- `schema` {GraphQLSchema} - `graphql` schema object
+- `document` {DocumentNode} - document query AST ,can be obtained by `parse` from `graphql`
+- `operationName` {string} - optional operation name in case the document contains multiple [operations](http://spec.graphql.org/draft/#sec-Language.Operations)(queries/mutations/subscription).
+- `compilerOptions` {Object} - Configurable options for the compiler
 
   - `disableLeafSerialization` {boolean, default: false} - disables leaf node serializers. The serializers validate the content of the field
```

Comment:
Explanation for this option is not clear. How is disabling leaf serialization related to validation?

Reply:
runtime validation of the output types is handled during the serialization of the field.